apps/prairielearn/src/question-servers/index.ts

```diff
@@ -21,7 +21,7 @@ const effectiveQuestionTypes = {
   Freeform: 'Freeform',
 } satisfies Record<NonNullable<QuestionType>, EffectiveQuestionType>;
 
-export function getEffectiveQuestionType(type: QuestionType): EffectiveQuestionType {
```

Comment:
Nit: the logic of this function could easily be moved into `getModule` below to simplify this code.